src/components/Link/Link.js

```diff
@@ -11,9 +12,14 @@ const Link = ({
   ...rest
 }) => {
   if (to.startsWith('/') || to.startsWith('#')) {
+    let internalUrl = new URL(to, siteMetadata.siteUrl);
```

Comment:
The second argument passed into `URL` here is:

1. Incorrect in environments other than production - eg. local development or Netlify preview environments.
2. Breaking hash URLs, such as those in [the right sidebar of docs](https://deploy-preview-209--roadie.netlify.app/docs/getting-started/getting-started-for-admins/) pages.

This needs more thought.